contactsummary/js/src/contact-summary/services/contact-details.service.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'common/moment'
+], function (_, moment) {
+  'use strict';
+
+  contactDetailsService.__name = 'contactDetailsService';
+  contactDetailsService.$inject = ['$q', '$log', 'apiService', 'modelService', 'settings'];
+
+  function contactDetailsService ($q, $log, Api, Model, settings) {
+    $log.debug('Service: contactDetailsService');
+
+    var factory = Model.createInstance();
```

Comment:
I found that we dont have `createInstance` function defined in any code. Only it is defined in `mock` files. Does that mean we are still using mocked data? Could you analyse why `Model.createInstance();` is necessary? (No need to look in details, if you can timebox 10-15 mins, it will be great.)

Reply:
It's here: https://github.com/civicrm/civihr/pull/2261/files/ad0360bf69e9f85586c57bd6066669d0bb776197#diff-a5856fd879a025de4ce2bb428f43991eR19